commands/blackjack.js

```diff
@@ -0,0 +1,199 @@
+const { SlashCommandBuilder, ButtonBuilder, ActionRowBuilder, ButtonStyle } = require('@discordjs/builders');
+const _ = require('lodash');
+const Deck = require('card-deck');
+const { DECK } = require('../constants');
+
+let deckOf52Cards;
+
+const convertAceToOne = (cards, cardSum) => {
+  for (var i = 0; i < cards.length; i++) {
+    if (cards[i].includes('A')) {
+      cards[i] = cards[i].replace('A', '1');
+      cardSum = cardSum - 10;
+      break;
+    }
+  }
+
+  return [cards, cardSum];
+};
+
+const getCardValue = (card) => {
+  let cardValue;
+  switch (card.substring(card.length - 1)) {
+    case 'T':
```

Comment:
what is T?

Reply:
T is 10. It was easier to parse as T rather than 10 since one character vs two characters.